__tests__/app.test.js

```diff
@@ -81,4 +81,52 @@ describe('/api/articles/:article_id', () => {
       })
     })
   })
-  
\ No newline at end of file
+  
+describe('/api/articles', () => {
+    test('GET: 200 should return an array of all articles,and all articles should have the following core properties: article_id,author,title,topic,created_at,votes,article_img_url', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toHaveLength(13)
+          body.articles.forEach((article) => {
+            expect(typeof article.article_id).toBe('number')
+            expect(typeof article.author).toBe('string')
+            expect(typeof article.title).toBe('string')
+            expect(typeof article.topic).toBe('string')
+            expect(typeof article.created_at).toBe('string')
+            expect(typeof article.votes).toBe('number')
+            expect(typeof article.article_img_url).toBe('string')
```

Comment:
You could neaten this up with an expect object containing matcher. That confirms the key and the data type.